config.go

```diff
@@ -467,8 +476,8 @@ type Config struct {
 
 	// Ghostferry requires a single numeric column to paginate over tables. Inferring that column is done in the following exact order:
 	// 1. Use the PerTable pagination column, if configured for a table. Fail if we cannot find this column in the table.
-	// 2. Use the FallbackColumn pagination column, if configured. Fail if we cannot find this column in the table.
-	// 3. Use the table's primary key column as the pagination column. Fail if the primary key is a composite key or is not numeric.
+	// 2. Use the table's primary key column as the pagination column. Fail if the primary key is not numeric or is a composite key without a FallbackColumn specified.
+	// 3. Use the FallbackColumn pagination column, if configured. Fail if we cannot find this column in the table.
```

Comment:
Can you update the code comment on L278 to reflect this new ordering?

Reply:
Good catch!